apps/citizen-func/package.json

```diff
@@ -43,7 +48,6 @@
     "express": "^4.15.3",
     "fp-ts": "^2.10.5",
     "io-ts": "^2.2.16",
-    "node-fetch": "^2.6.1",
```

Comment:
Are we using the native fetch now?

Reply:
No, I just found that we were not using `node-fetch` anywhere... :'(